stacks/data/generate/data_workloads.py

```diff
@@ -47,11 +61,13 @@ def render_template_components(config: WorkloadConfigBaseModel, template_source_
     template_list = template_env.list_templates(extensions=".jinja")
     for template in template_list:
         template = template_env.get_template(template)
-        template_filepath = Path(template.filename.split(template_source_path, 1)[1])
+        template_filepath = Path(template.filename.split(str(template_source_path), 1)[1])
```

Comment:
I don't think we should have to convert this to a string here, as the function already specifying str type for this argument.

Reply:
You would think that wouldn't you.  Doesn't work without it.

Reply:
This suggests that we're not actually passing the `template_source_path` argument as a string... in that case it would be better to do the conversion before we call this function (within `generate_pipeline`).